client/state/jetpack-onboarding/actions.js

```diff
@@ -37,6 +38,12 @@ export const saveJetpackOnboardingSettings = ( siteId, settings ) => ( {
 	},
 } );
 
+export const saveJetpackOnboardingSettingsSuccess = ( siteId, settings ) => ( {
```

Comment:
Maybe add a comment above this method that explains [why it is needed](https://github.com/Automattic/wp-calypso/pull/22131#issuecomment-363132972) even tho it doesn't seem to do anything, so people (e.g. future me) aren't tempted to _Rip it All Out™️_

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/22131/commits/baf9ca00995fdd7fb03b8bf0146f5d06e756b50b - let me know if that looks good.

Reply:
Looks great! Maybe say `progress indicator` instead of just `progress`.